drivers/sensor/bosch/bma4xx/bma4xx_emul.c

```diff
@@ -203,7 +203,7 @@ void bma4xx_emul_set_accel_data(const struct emul *target, q31_t value, int8_t s
 	int16_t reg_val;
 
 	/* 0x00 -> +/-2g; 0x01 -> +/-4g; 0x02 -> +/-8g; 0x03 -> +/- 16g; */
-	int64_t accel_range = (2 << data->regs[BMA4XX_REG_ACCEL_RANGE]);
+	int64_t accel_range = ((int64_t)2 << data->regs[BMA4XX_REG_ACCEL_RANGE]);
```

Comment:
I think the more idiomatic way would be as follows, no? And you can prob get rid of the unnecessary parentheses while you're here 

```suggestion
	int64_t accel_range = 2LL << data->regs[BMA4XX_REG_ACCEL_RANGE];
```